networkx/generators/lattice.py

```diff
@@ -145,25 +145,32 @@ def grid_graph(dim, periodic=False):
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def hypercube_graph(n):
-    """Returns the *n*-dimensional hypercube graph.
-
-    The nodes are the integers between 0 and ``2 ** n - 1``, inclusive.
-
-    For more information on the hypercube graph, see the Wikipedia
-    article `Hypercube graph`_.
+    """
+    Return the *n*-dimensional hypercube graph.
```

Comment:
Let's keep this matching the first line grammar and whitespace of the other functions in this module.
```suggestion
    """Returns the *n*-dimensional hypercube graph.
```